### PR TITLE
Feature/eodhp 10 deploy cms to argocd

### PR DESCRIPTION
**Pull request by hcollingwood**
Web presence for the EODHP

Includes:
- Updates to ArgoCD configuration to include web presence

### CODE REVIEW

File: apps/web-presence/values.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+app_url: dev.eodhp.eco-ke-staging.com
+
+database:
+  sql_user: "admin"
+  sql_password: "password"
+  number_of_replicas: 1
```
Comment (sgillies-tpzuk) [inline apps/web-presence/values.yaml:6]: i removed this option from the helm chart for now, so its clear it cannot be modified. Can you remove please.

File: eodhp/apps.yaml [hunk 2]
```diff
@@ -7,7 +7,7 @@ spec:
   generators:
     - git:
         repoURL: https://github.com/UKEODHP/eodhp-argocd-deployment.git
-        revision: HEAD
+        revision: feature/EODHP-10-deploy-cms-to-argocd
```
Comment (sgillies-tpzuk) [inline eodhp/apps.yaml:10]: remember to change these back to HEAD before you merge, but don't do it before PR is accepted.

**Review by sgillies-tpzuk — CHANGES_REQUESTED** [review]
Just one change required and we can close this monster off!

File: apps/web-presence/values.yaml (hunk 1 above)
Comment (hcollingwood) [reply apps/web-presence/values.yaml]: Removed

**Review by sgillies-tpzuk — APPROVED** [review]
All good. A final reminder to point argocd back to HEAD before merging :)

**Pull request merged**